events.go

```diff
@@ -1,5 +1,8 @@
 package scalingo
 
+//go:generate go run cmd/gen_events_boilerplate/main.go
+//go:generate go run cmd/gen_events_specialize/main.go
```

Comment:
I don't understand the purpose of this command. I assume it's a Go instruction thing but I don't know what they are.

Reply:
Basically it says:

When `go generate` is run, run the following command, so just running some generators which were added to the project